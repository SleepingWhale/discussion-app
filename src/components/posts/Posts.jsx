import React from 'react';

import styles from './Posts.module.css';
import { SearchContainer } from '../search';
import { Post } from '../post';
import { PostEditorContainer } from '../postEditor';

export function Posts({ posts }) {
  return (
    <div className={styles.container}>
      <div className={styles.search}>
        <SearchContainer />
      </div>
      <PostEditorContainer />
      {posts.map(p => (
        <Post {...p} key={p.id} />
      ))}
    </div>
  );
}
