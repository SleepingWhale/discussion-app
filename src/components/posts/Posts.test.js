import React from 'react';
import { shallow } from 'enzyme';
import { Posts } from './Posts';

describe('Posts', () => {
  it('should render correctly', () => {
    const component = shallow(<Posts posts={[]} />);
    expect(component).toMatchSnapshot();
  });
});
