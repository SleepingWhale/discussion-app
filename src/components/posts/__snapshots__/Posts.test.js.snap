// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Posts should render correctly 1`] = `
<div
  className="container"
>
  <div
    className="search"
  >
    <ConnectFunction />
  </div>
  <ConnectFunction />
</div>
`;
